### ISSUE TITLE
[BC Break] Unify naming of tag attributes

### ISSUE THREAD

**Issue by mvar** [Sylius/Sylius#7838]
Current Sylius configuration:

```yml
services:
    foo.grid.filter.bar:
        class: FooBundle\Grid\Filter\BarFilter
        tags:
            -
                name: sylius.grid_filter
                type: bar
                form-type: 'FooBundle\Form\Type\Filter\BarType'
```

Please note `form-type`.

In Symfony all attributes follows underscore naming convention. As Sylius is highly dependent on Symfony, it would be nice to see the same coding style too. I suggest to find and replace all similar places before first RC.

**Comment by mvar** [comment]
@pjedrzejewski won't happen?

**Comment by lchrusciel** [comment]
@mvar are you willing to open a PR with this fix?

**Comment by mvar** [comment]
@lchrusciel I have opened PR with updated code and docs. I haven't added any notes to UPGRADE doc, I'm not sure if I should.

**Comment by pamil** [comment]
Merged in #8096. :)
